### PR TITLE
Bump debian pg_cron to 1.6.0

### PR DESCRIPTION
**Pull request by gurkanindibay**

### CODE REVIEW

File: postgres-matrix.yml [hunk 1]
```diff
@@ -6,3 +6,5 @@ version_matrix:
       postgres_versions: [10, 11, 12, 13, 14]
   - 1.4.2:
       postgres_versions: [ 10, 11, 12, 13, 14, 15 ]
+  - 1.5.2:
```
Comment (hanefi) [inline postgres-matrix.yml]: Should this be 1.6 instead?

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,10 @@
+pgcron (1.6.0.citus-1) stable; urgency=low
+
+  * Official 1.6.0 release of PG Cron
+  * Adds support for PostgreSQL 16
+
```
Comment (hanefi) [inline debian/changelog:5]: We did not use to have an empty line here for earlier versions.

**Review by hanefi — CHANGES_REQUESTED** [review]

File: debian/changelog (hunk 2 above)
Comment (gurkanindibay) [reply debian/changelog:5]: All of them are standard. Do I miss sth?

File: postgres-matrix.yml (hunk 1 above)
Comment (gurkanindibay) [reply postgres-matrix.yml]: Yes Fixed. thanks

File: debian/changelog (hunk 2 above)
Comment (hanefi) [reply debian/changelog:5]: It appears that my browser was rendering a strange editor view. Your changes are totally fine in this part of the file.

**Review by hanefi — APPROVED** [review]

**Pull request merged**